introduction.md

```diff
@@ -0,0 +1 @@
+I'm super passionate about both front-end development 👨🏻‍💻 and design 👨🏻‍🎨 - especially the vectors where these two fields meet 🦄!
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.